include/dronecore.h

```diff
@@ -34,6 +35,8 @@ class DroneCore
 
     ConnectionResult add_udp_connection();
     ConnectionResult add_udp_connection(int local_port_number);
+    ConnectionResult add_tcp_connection();
```

Comment:
@julianoes When you've worked out the doc standard, can you please make sure it gets added to this. Specifically, the default ip and port need to be documented.  Also if there is any internal timeout, what that is, and whether it can be configured.